packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -205,11 +218,43 @@ export class CXLMarketingNavElement extends LitElement {
       }, 1000)
     );
 
-    document.body.addEventListener('vaadin-overlay-open', this._onOverlayOpen.bind(this));
+    document.body.addEventListener('vaadin-overlay-open', this._boundOnOverlayOpen);
 
     super.firstUpdated(changedProperties);
   }
 
+  /*
+  * Setup listeners on nav items to handle overlays events.
+  */
+  _setupSlottedMenuItems () {
```

Comment:
This fixes the bug discussed [here](https://app.slack.com/client/T01HJPKP9TQ/C01HXUNGEPM/thread/C01HXUNGEPM-1692190828.079959).  Credit to @anoblet for finding the solution.